sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -18,6 +18,9 @@
 package org.apache.spark.sql
 
 import java.io.CharArrayWriter
+import java.util.Random
+
+import org.apache.spark.util.random.SamplingUtils
```

Comment:
(it seems we need to reorder imports, https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Imports)